# How to cite

BridgeDb is a framework for finding and mapping equivalent identifiers from various databases.
BridgeDb has many facets: providing a framework, live services, and identifier mapping files
for genes, gene-variant, proteins, metabolites and interactions. BridgeDb is a 100%
Open Science project. The software is available under an Apache License 2.0 and identifier
mapping databases under Open licenses, CCZero where possible (e.g. for the metabolite
identifier mapping databases). 

When using the framework of BridgeDb, webservice or implementation thereof, please cite:

 * [The BridgeDb framework: standardized access to gene, protein and metabolite identifier mapping services](https://doi.org/10.1186/1471-2105-11-5). van Iersel MP, Pico AR, Kelder T, Gao J, Ho I, Hanspers K, Conklin BR, Evelo CT. BMC Bioinformatics. 2010 Jan 4;11(1):5.

When discussing the use of BridgeDb to link chemical data or toxicology, please cite:

 * [Scientific lenses to support multiple views over linked chemistry data](https://dx.doi.org/10.1007/978-3-319-11964-9_7). Batchelor C, Brenninkmeijer C, Chichester C, Davies M, Digles D, Dunlop I, Evelo CT, Gaulton A, Goble C, Gray AJG, Groth P, Harland L, Karapetyan K, Loizou A, Overington JP, Pettifer S, Steele J, Stevens R, Tkachenko V, Waagmeester A, Williams A, Willighagen EL. In The Semantic Web – ISWC 2014, Lecture Notes in Computer Science Volume 8796, 2014, 98-113.
 * [The future of metabolomics in ELIXIR](https://f1000research.com/articles/6-1649/v2). Van Rijswijk M, Beirnaert C, Caron C, Cascante M, Dominguez V, Dunn WB, Ebbels TM, Giacomoni F, Gonzalez-Beltran A, Hankemeier T, Haug K, Izquierdo-Garcia JL, Jimenez RC, Jourdan F, Kale N, Klapa MI, Kohlbacher O, Koort K, Kultima K, Le Corguillé G, Moreno P, Moschonas NK, Neumann S, O’Donovan C, Reczko M, Rocca-Serra P, Rosato A, Salek RM, Sansone SA, Satagopam V, Schober D, Shimmo R, Spicer RA, Spjuth O, Thévenot EA, Viant MR, Weber RJM, Willighagen EL, Zanetti G, Steinbeck C. F1000Research, 6 (2017) .  [version 2; referees: 3 approved]
 * [Introducing WikiPathways as a data-source to support Adverse Outcome Pathways for regulatory risk assessment of chemicals and nanomaterials](https://www.frontiersin.org/articles/10.3389/fgene.2018.00661/abstract). Martens M,  Verbruggen T, Nymark P, Grafström R, Burgoon LD, Aladjov H, Torres Andón F, Evelo CT, Willighagen EL. Front. Genet. (2018).

When using specific plugin or use case of BridgeDb (related to Cytoscape, and the new mapping files on gene-variants), please cite:

 * [CyTargetLinker: A Cytoscape App to Integrate Regulatory Interactions in Network Analysis](https://doi.org/10.1371/journal.pone.0082160). Kutmon M, Kelder T, Mandaviya P, Evelo CTA, Coort SL. PloS one 8.12 (2013): e82160.
 * [WikiPathways App for Cytoscape: Making biological pathways amenable to network analysis and visualization](https://dx.doi.org/10.12688%2Ff1000research.4254.1). Kutmon M, Lotia S, Evelo CT, Pico AR.
 * [CyTargetLinker app update: A flexible solution for network extension in Cytoscape](https://doi.org/10.12688/f1000research.14613.2). Kutmon M, Ehrhart F, Willighagen EL, Evelo CT, Coort SL. [version 2; peer review: 2 approved]. F1000Research 2019, 7:743
 * Pre-Print: [Providing gene-to-variant and variant-to-gene database identifier mappings to use with BridgeDb mapping services](https://f1000research.com/articles/7-1390/v1). Ehrhart F, Melius J, Cirillo E, Kutmon M, Willighagen EL, Coort SL, Curfs LMG, Evelo CT. F1000Research 7 (2018) [version 1: awaiting peer review].
