# Development

The development of BridgeDb is as modular as BridgeDb itself. In particular, these teams exist:

1. [Java library developers](https://github.com/bridgedb/BridgeDb)
2. Creators of the identifier mapping databases
   1. [Genes and Proteins (Based on Ensembl & EnsemblGenomes)](https://github.com/bridgedb/BioMartScript)
   2. [Metabolites (Based on HMDB, ChEBI, and Wikidata)](https://github.com/bridgedb/create-bridgedb-metabolites)
   3. [Interactions (Based on Rhea)](https://github.com/bridgedb/InteractionDB)
   4. [Gene-Variant (Ensembl and dnSNP)](https://github.com/BiGCAT-UM/BridgeDbVariantDatabase)

## Development team

The following groups (in alphabetical order for groups and people) are contributing to the development of the BridgeDb world (current or past):

 * Gladstone Institutes: Alex Pico, Anders Riutta
 * Heriot Watt University: Alasdair Gray
 * Maastricht University: Anwesha Bohler, Chris Evelo, Martijn van Iersel, Thomas Kelder, Christ Leemans, Marvin Martens, Luc, Meyer Jonathan MÃ©lius, Denise Slenter,  Lucas Uberti-Bona Marin, Egon Willighagen
 * University of Manchester: Christian Brenninkmeijer, Ian Dunlop, Carole Goble, Nick Juty, Stian Soiland-Reyes
 * Others:  Jianjiong Gao, Isaac Ho, Manas Awasthi

## Funding and sustainability

BridgeDb has been funded over the years as part of many projects, including: the Open PHACTS project, OpenRiskNet,
EU-ToxRisk, NanoCommons, ELIXIR Implementation Studies MolData1 and MolData2 (2016 and 2017) and
Metabolomics (2018), and FAIRplus. 

Sustainability is further ensured by the Open Science nature and the continued dependence of other projects on
BridgeDb. Furthermore, people involved continuously push a standardized, FAIR identifier mapping database in
other data integration projects, encouraging broader sustainability.
